app/controllers/sites/rss_feeds_controller.rb

```diff
@@ -70,7 +70,7 @@ def assign_rss_feed_urls(attributes)
     existing_rss_feed_urls = []
     new_urls = []
 
-    rss_feed_urls_attributes = attributes || {}
+    rss_feed_urls_attributes = attributes.to_h || {}
```

Comment:
This fixes the issue on the line identified by the deprecation warning, but it would be preferable to convert the params to a hash in the method that extracts the params:
```ruby
  def rss_feed_params
    params.require(:rss_feed).permit(:name,
                                     :show_only_media_content,
                                     { rss_feed_urls_attributes: [:url] }).to_h
  end
```
That will ensure that the `rss_feed_params` are already a hash whenever they are used in the controller.